Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -291,18 +291,24 @@ def condition_(self, fragment, stepList, key, hasHarvest):
 
 class UpgradeWorkflow_trackingMkFit(UpgradeWorkflowTracking):
     def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Digi' in step: stepDict[stepName][k] = merge([self.step2, stepDict[step][k]])
         if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
     def condition_(self, fragment, stepList, key, hasHarvest):
         return '2017' in key or '2021' in key
 upgradeWFs['trackingMkFit'] = UpgradeWorkflow_trackingMkFit(
     steps = [
+        'Digi',
+        'DigiTrigger',
         'Reco',
         'RecoGlobal',
     ],
     PU = [],
     suffix = '_trackingMkFit',
     offset = 0.7,
 )
+upgradeWFs['trackingMkFit'].step2 = {
```

Comment:
are you going to make use of the mkFit at HLT as well ? I'm confused :(

Reply:
This customize function provides a way to test mkFit in HLT menu, regardless of whether it will be used there. Having the customize function in the matrix workflow helps to keep the customize function functional.

Written that, I don't feel strongly on this point, so if you think it would be easier to not have the HLT customization, I can remove it.

Reply:
I am not sure we really know yet what/if some level of usage in an adapted mkFit for HLT will be needed. So not sure. 
Maybe we revisit this when we know more?